deployments/run.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Script used for running and updating bridge deployments.
+#
+# To deploy a network you can run this script with the name of the network you want to run.
+#
+# `./run.sh eth-poa-sub`
+#
+# To update a deployment to use the latest images available from the Docker Hub add the `update`
+# argument after the bridge name.
+#
+# `./run.sh rialto-millau update`
+#
+# Once you've stopped having fun with your deployment you can take it down with:
+#
+# `./run.sh rialto-millau stop`
+
+set -xeu
+
+RIALTO=' -f ./networks/docker-compose.rialto.yml'
```

Comment:
```suggestion
RIALTO=' -f ./networks/rialto.yml'
```
IMHO this stuff is redundant, the deployment-level README explains that these files are docker-compose files, we've touch on this off-band, but I'm still not convinced what the value of adding this redundancy is :)

Reply:
All the docs I've read on override files have the prefix, but since we basically only run these with the `-f` flag anyways it doesn't matter. I've removed the prefix from the names

Reply:
> All the docs I've read on override files have the prefix

It's not really an override, we compose a file from multiple other files - `-f networks/rialto.yml` adds a bunch of unique services, so is `-f networks/millau.yml`. `bridges/` stuff is more like an override.